### DISCUSSION TITLE
Netdiscover Issue on UTM on Mac: Ubuntu Emulation Not Appearing in Virtual Machine Setup

### DISCUSSION THREAD

**Discussion by arpkk** [utmapp/UTM#6251] [Linux]
hello, the situation is as follows: for an exercise in an ethical hacking course, I am setting up a virtual machine Kali in bridge mode on UTM for MAC m2 (from where the attack will be carried out) and an emulation of Ubuntu. The problem is that when I use the netdiscover command to see which devices are connected to the Wi-Fi network, the Ubuntu emulation does not appear.

I followed this tutorial https://patrick-rottlaender.medium.com/create-a-virtual-hacking-lab-on-apple-silicon-mac-a86d9b3b2e5f and he uses "Host Only" instead of bridged and it didn't work for me either. I tried changing "Bridge Interface" to en0, I tried changing the emulated network card, and nothing worked.

**Configuration**  
* UTM Version: 4.4.5
* macOS Version: Sonoma 14.4.1
* Mac Chip: M2 

![imagen_2024-04-04_131426241](https://github.com/utmapp/UTM/assets/65407043/9baf19ed-ee23-4ac5-92d4-75dd3d8b16e3)
![imagen_2024-04-04_131434718](https://github.com/utmapp/UTM/assets/65407043/a24f800a-473d-4026-b7dc-218cb9b05db7)
[config.zip](https://github.com/utmapp/UTM/files/14874093/config.zip)

**Comment by m4th3r0b0t** [comment]
Having the same exact issues. Kindly let me know if you find any solution.
